.github/CODEOWNERS

```diff
@@ -74,3 +74,10 @@
 # ./lib/internal/modules/* @nodejs/modules
 # ./lib/internal/bootstrap/loaders.js @nodejs/modules
 # ./src/module_wrap* @nodejs/modules @nodejs/vm
+
+# N-API
+
+# ./src/node_api* @nodejs/n-api
```

Comment:
are these supposed to be commented out

Reply:
@devsnek that is a good question. I saw most of them were commented out so followed the same pattern.  @jasnell since you left many of the them commented out for now is it ok to turn it on for N-API now or do you think we should let the initial set bake for a while? Either way I'd like to add N-API now (either active or commented out) so that we don't forget.

Reply:
The ones commented out are for teams which does not have write permission on this repository (teams which are not subteams of collaborators), as per discussions in https://github.com/nodejs/node/pull/33895.

I'm working on an alternative to codeowners using the github-bot to work around GitHub's constraint on only teams with write permissions being valid owners. IMO it's best to keep these commented out for now, then we can uncomment when we move to the github-bot-based owners. Note that this PR is essentially a no-op this way though.